### ISSUE TITLE
Implement log rotation

### ISSUE THREAD

**Issue by sauloperez** [openfoodfoundation/ofn-install#96]
Implement proper log rotation as discussed in https://github.com/openfoodfoundation/openfoodnetwork/pull/1927

**Comment by mkllnk** [comment]
I added this to our Git post-receive hook which we use to deploy:
```sh
# Rotate log files
logrotate -s "$CURRENT_PATH/log/logrotate-status" "$CURRENT_PATH/log/logrotate.conf"
```

**Comment by sauloperez** [comment]
I'd be great to know the contents of the logrotate.conf as well.

**Comment by mkllnk** [comment]
Ah, sure. Sorry. Here it is:
```
/home/openfoodnetwork/apps/openfoodnetwork/current/log/*.log {
        missingok
        compress
        rotate 5
	weekly
}
```

So I decided to do it on deploy so that the log files are not cut in the middle. But it has the disadvantage that they can grow until the disk is full if you never deploy.

It rotates only if the file is older than a week so that we keep at least the last five weeks of log files.

We could set this up with cron and have size restrictions. But which values are useful depends on how much disk space is free for logs. So I just kept it minimal.
